<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaVendas</class>
 <widget class="QDialog" name="fm_gestaVendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listavenda">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>561</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaProdutoVenda">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>561</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_geraPDF">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar Relat√≥rio PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_filtar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tnn_mostratudo">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostar todas as vendas</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
